versions.tf

```diff
@@ -19,16 +19,31 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 5.0"
+      version = ">= 4.66"
+    }
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = "~> 4.70"
+    }
+    archive = {
+      source  = "hashicorp/archive"
+      version = ">= 2"
+    }
+    null = {
+      source  = "hashicorp/null"
+      version = "~> 3.2"
+    }
+    time = {
+      source  = "hashicorp/time"
+      version = "~> 0.9.1"
     }
   }
 
   provider_meta "google" {
     module_name = "blueprints/terraform/gen-ai-document-summarization/v0.0.1"
   }
+}
 
-  backend "gcs" {
-    bucket  = null
-    prefix  = null
-  }
+provider "google" {
+  project = var.project_id
```

Comment:
Let's configure this per resource rather than configuring the provider

Reply:
Thanks for catching this. Updated.